modules/testkit/src/test/scala/com/caesars/tracing/CompletedSpanQueue.scala

```diff
@@ -0,0 +1,22 @@
+package com.caesars.tracing
+
+import cats.effect.Ref
+import io.janstenpickle.trace4cats.model.CompletedSpan
+import zio.Task
+import zio.interop.catz.asyncRuntimeInstance
+import zio.interop.catz.implicits.rts
+
+import scala.collection.immutable.{Queue => ScalaQueue}
```

Comment:
minor points stylistic points here:
1. use `as` instead of `=>` here since we're adhering to scala 3 syntax by default.
2. I'd alias this as StdQueue (Std as in standard library) instead of ScalaQueue. Everything here is scala, including the zio Queue, and giving a nod to the standard library yields more specific/accurate naming (without being unnecessarily verbose).